### ISSUE TITLE
Question about loss

### ISSUE THREAD

**Issue by ChrisLLLLL** [mkhorr/supergluetraining#3]
Hello,thanks for your code. when i try use it .  i run train.py 500epoch.   But the loss still at 300. Could you help me analyze the reason for this situation？ Looking forword to your reply!

**Comment by mkhorr** [comment]
Hello, In my experience, a loss of 300 after 500 epochs sounds about right. Have you tried comparing the evaluation metrics with that of the pretrained model? These metrics should continue to approach the pretrained model's metrics despite the loss appearing stagnant.

**Comment by ChrisLLLLL** [comment]
Thank you for your answer. I just test it on some pictures. The result is ok, but a bit worse than the pre-training. And, there are a lot of blue lines(less confident). May be i used less scannet dataset. I want to know, do you have a better way to download the .sens dataset, I can only download one by one.  
For a better result, do I need to reduce the loss? how can i do for this. need to set more epochs to train or other methods?

**Comment by mkhorr** [comment]
The default parameters (e.g. 500 epochs) were set based on the original paper and with the expectation that the entire scannet dataset is used. The training code can still be used with less data, but will likely require more epochs for reasonable results. As for the .sens files, you could write a script to automate the downloading process if you have the urls.

**Comment by ChrisLLLLL** [comment]
Thanks a lot. That makes sense to me.
